pkg/issuer/acme/dns/clouddns/clouddns_test.go

```diff
@@ -101,9 +101,46 @@ func TestLiveGoogleCloudCleanUp(t *testing.T) {
 
 	time.Sleep(time.Second * 1)
 
-	provider, err := NewDNSProviderCredentials(gcloudProject, util.RecursiveNameservers)
+	provider, err := NewDNSProviderCredentials(gcloudProject, util.RecursiveNameservers, "")
 	assert.NoError(t, err)
 
 	err = provider.CleanUp(gcloudDomain, "_acme-challenge."+gcloudDomain+".", "123d==")
 	assert.NoError(t, err)
 }
+
+func TestDNSProvider_getHostedZone(t *testing.T) {
+	testProvicer, err := NewDNSProviderCredentials("my-project", util.RecursiveNameservers, "test-zone")
```

Comment:
```suggestion
	testProvider, err := NewDNSProviderCredentials("my-project", util.RecursiveNameservers, "test-zone")
```

It'll need changing below too.